src/test/kotlin/io/github/mojira/arisa/modules/HelpersTest.kt

```diff
@@ -0,0 +1,267 @@
+    }
+
+    "assertNull when given null should return ModuleResponse in Either" {
+        val moduleEither = assertNull(null)
+
+        moduleEither shouldBeRight ModuleResponse
+    }
+
+    "assertNull when given not null should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertNull(1337)
+
+        moduleEither shouldBeLeft OperationNotNeededModuleResponse
+    }
+
+    "assertNotNull when given not null should return ModuleResponse in Either" {
+        val moduleEither = assertNotNull(1337)
+
+        moduleEither shouldBeRight ModuleResponse
+    }
+
+    "assertNotNull when given null should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertNotNull(null)
+
+        moduleEither shouldBeLeft OperationNotNeededModuleResponse
+    }
+
+    "assertEither when given arguments one of which returns ModuleResponse in Either it should return ModuleResponse in Either" {
+        val moduleEither = assertEither(OperationNotNeededModuleResponse.left(), ModuleResponse.right())
+
+        moduleEither shouldBeRight ModuleResponse
+    }
+
+    "assertEither when given arguments none of which return ModuleResponse in Either it should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertEither(OperationNotNeededModuleResponse.left(), OperationNotNeededModuleResponse.left())
+
+        moduleEither shouldBeLeft OperationNotNeededModuleResponse
+    }
+
+    "tryRunAll when given functions that run successfully should run all of them and return ModuleResponse in Either" {
+        var checker = ""
+        fun successful1(): Either<Throwable, Unit> {
+            checker += "a"
+            return Unit.right()
+        }
+        fun successful2(): Either<Throwable, Unit> {
+            checker += "b"
+            return Unit.right()
+        }
+        fun successful3(): Either<Throwable, Unit> {
+            checker += "c"
+            return Unit.right()
+        }
+        val moduleEither = tryRunAll(listOf({successful1()}, {successful2()}, {successful3()}))
+
+        moduleEither shouldBeRight ModuleResponse
+        checker shouldBe("abc")
+    }
+
+    "tryRunAll when given functions that run successfully and one that throws should run all of the ones before the Throwable and return FailedModuleResponse in Either" {
+        var checker = ""
+        fun successful1(): Either<Throwable, Unit> {
+            checker += "a"
+            return Unit.right()
+        }
+        fun successful2(): Either<Throwable, Unit> {
+            checker += "b"
+            return Unit.right()
+        }
+        fun successful3(): Either<Throwable, Unit> {
+            checker += "c"
+            return Unit.right()
+        }
+        val moduleEither = tryRunAll(listOf({successful1()}, {successful2()}, {throwable.left()},
+            {Throwable(message="different error").left()}, {successful3()}))
```

Comment:
```suggestion
            { Throwable(message = "different error").left() }, { successful3() }))
```
Last formatting error

Reply:
fixed by https://github.com/mojira/arisa-kt/pull/433/commits/198d97a8e7f4326a4a5fa11a8e6ae7b5988c07f2

Reply:
ah there's a bit more k

Reply:
fixed by https://github.com/mojira/arisa-kt/pull/433/commits/4dabc044716fe33694ff7843f0107a7e543c883f